public/config/configuration.json

```diff
@@ -7,5 +7,5 @@
   "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "KEYCLOAK_CONFIG_PATH": "/local-keycloak-config-url/keycloak.json",
   "ADDRESS_COMPLETE_KEY": "RH99-RN99-GB72-FW86",
-  "LD_CLIENT_ID": "5dc1c5fd1d213d087f9e1024"
+  "LD_CLIENT_ID": "5e5d5af4574f7e09ab53d03b"
```

Comment:
Discussed with @lekshmimallika-aot and we think we probably should use a single LD_CLIENT_ID so we can disable the features in one location. Thoughts?

Reply:
Sure, but what about the different environments?

And if you change it here, do you also have to change the config map keys for Dev and/or Test and/or Prod?

Reply:
I am not sure if we have different client ids per environment, @lekshmimallika-aot ?

But yes, if we change it here we need to change it for dev, test, and prod so it shares the same set of properties

Reply:
Different environments will have different client ID. That will have to be updated in the config map of the respective environments. Here we can keep this one